src/main/java/org/b612foundation/adam/datamodel/EntityFactory.java

```diff
@@ -0,0 +1,7 @@
+package org.b612foundation.adam.datamodel;
```

Comment:
Do we need this file at this point? Maybe deal with it later, if we don't replace it with something else entirely?

Reply:
I don't think so, just wasn't sure what we wanted to do. Happy to take it out now or wait.

Reply:
Let's remove it from this PR at least.

Reply:
Removed.